packages/acceptance-tests/pkg-tests-specs/sources/commands/constraints/__snapshots__/check.test.js.snap

```diff
@@ -38,6 +38,17 @@ Object {
 }
 `;
 
+exports[`Commands constraints check test (empty project / gen_invalid_dependency (deep field path)) 1`] = `
+Object {
+  "code": 1,
+  "stderr": "",
+  "stdout": "➤ BR0001: Error: uncaught exception: error(existence_error(procedure, '/'(workspace_has_dependency, 4)), '/'(gen_invalid_dependency, 4))
```

Comment:
Hm that's not expected 😃

Reply:
Might require a rebase against master to get `DependencyType`

Reply:
This happens in the empty workspace, because our prolog rules don't contain any `workspace_has_dependency/4` predicates. We should probably add a

```prolog
workspace_has_dependency(_, _, _, _):- false.
```

at the end of the database to prevent this.